data/jobs.txt

```diff
@@ -1573,3 +1587,114 @@ mission "Large Rush Delivery [3]"
 		payment
 		payment 42000
 		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>."
+		
+mission "Wealthy Executive Short [0]"
+	name "Transport a wealthy executive  to <planet>"
+	job
+	repeat
+	deadline 0 1
+	description "Bring a wealthy and important executive to <destination> by <date>. He is in a hurry so you will not have time to stop to refuel."
+	to offer
+		random < 10
+		or 
+			has "ship: Flivver"
```

Comment:
instead of hardcoding specific ships here, perhaps add new ship category "Luxury Transport" and move these ships to that category?